composer.json

```diff
@@ -18,6 +18,7 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "scripts": {
-        "lint": "vendor/squizlabs/php_codesniffer/bin/phpcs --report=summary,source"
+        "lint": "vendor/squizlabs/php_codesniffer/bin/phpcs --report=summary,source",
+	"test": "vendor/bin/phpunit"
```

Comment:
Indentation is mixed here. Tab is used for the 22nd line.
phpcs path can be shortened to `vendor/bin/phpcs` as well.

Reply:
> Hi @raaaahman, about the PHPCS check, I reccommend using a PHPCS action for annotation support.

That could be pretty neat indeed!